.azuredevops/build-and-push-prod.yml

```diff
@@ -4,18 +4,41 @@ trigger:
 pr: none
 
 variables:
-  azureContainerRegistry.repository: 'eligibilityestimator'
+  azureContainerRegistry.repository: 'oas/eligibility-estimator'
```

Comment:
DTSShared still has a repository called "eligibilityestimator". Will that change to 'oas/eligibility-estimator' or a new one for prod will be created? Not sure how this part works.